implementations/rust/ockam/ockam_identity/src/secure_channel/encryptor.rs

```diff
@@ -70,11 +70,10 @@ impl Encryptor {
             .aead_encrypt(&self.key, payload, &nonce, &[])
```

Comment:
Let's actually pass the destination buffer to this function to avoid another allocation